readme.md

```diff
@@ -32,7 +32,7 @@ setInterval(() => {
 
 #### React via useSnapshot
 
-Create a local snapshot that catches changes. Rule of thumb: read from snapshots, mutate the source. The component will only re-render when the parts of the state you access have changed, it is render-optimized.
+Create a local snapshot that catches changes. Rule of thumb: read from snapshots in React render function, otherwise use the source. The component will only re-render when the parts of the state you access have changed, it is render-optimized.
```

Comment:
Let's make it simpler.
```suggestion
Create a local snapshot that catches changes. Rule of thumb: read from snapshots in render function, otherwise use the source. The component will only re-render when the parts of the state you access have changed, it is render-optimized.
```